scripts/core/ui/components/index.ts

```diff
@@ -25,6 +25,7 @@ import {Dropdown} from './dropdown';
 import {DropdownMenuDivider} from './dropdown-menu-divider';
 import {NavTabs} from './nav-tabs';
 import {DropZone} from './drop-zone';
+import {PlainTextEditor} from './PlainTextEditor/PlainTextEditor';
```

Comment:
Why are you not importing it directly?

Reply:
what do you mean?

Reply:
You are importing `PlainTextEditor`, are not doing anything with it, and exporting it. Why not import it directly in the file where you are about to use it?